docs/getting-started.md

```diff
@@ -117,7 +117,7 @@ tar.exe xvf .\containerd-windows-amd64.tar.gz
 
 # Copy and configure
 Copy-Item -Path ".\bin\" -Destination "$Env:ProgramFiles\containerd" -Recurse -Force
```

Comment:
So I think the reason the instructions might not work in some cases is because if the destination directory already exists it will create a subdirectory and copy the contents of the source recursively. 

For example:
If $Env:ProgramFiles\containerd doesn't exist already (the containerd subdirectory specifically as I'd hope programfiles does haha) the directory will be created and then the contents of .\bin will be plopped inside.

If the $Env:ProgramFiles\containerd directory already exists, instead a bin subdirectory will be created inside and the contents of .\bin will be placed in $Env:ProgramFiles\containerd\bin. 

So that's probably what's happening here, the authors machine likely already has that directory present so the instructions don't work out the box.

Reply:
We could change these to create the directory first and then `Copy-Item -Path ".\bin\*" -Destination "$Env:ProgramFiles\containerd" -Recurse -Force`

Reply:
what are the final thoughts ? I can put @dcantah's idea then try it. By the way In my case, the destination directory was not exist.

Reply:
If the destination didn't exist and the instructions didn't work I'm not sure what could be happening :/. I've tried this on two separate machines now with success. I don't think making it more robust in case someone does have the dir there is bad, but it is odd if these don't work in the case where that's not true

Reply:
How about taking an extra step to make sure the dir exists before making the copy? e.g.
```
New-Item -Path "$Env:ProgramFiles\containerd\bin" -Type Directory -Force
Copy-Item -Path ".\bin\*" -Destination "$Env:ProgramFiles\containerd\bin" -Force
```